### DISCUSSION TITLE
I used to have a 'Re-run all jobs' button in my GitHub runner, but it seems to have disappeared. I would appreciate any help in finding out how to get this button back.

### DISCUSSION THREAD

**Discussion by ansiace** [community/community#138101] [Actions] [answered]
Help me out

**Comment by mdazfar2** [comment]
> It is recommended to write the main issue in body not title

GitHub recently made some changes to the Actions UI, which removed the 'Re-run all jobs' button from the runner page. Instead, you can now re-run all jobs from the Actions tab in your repository.

Here's how to re-run all jobs from the Actions tab:

1. Go to your repository on GitHub.
2. Click on the "Actions" tab.
3. Select the workflow run you want to re-run all jobs for.
4. Click on the three dots in the top-right corner of the workflow run and select "Re-run all jobs" from the dropdown menu.

This will trigger a new workflow run with all jobs re-run.

Alternatively, if you want to re-run a specific job, you can do so by following these steps:

1. Go to the workflow run you want to re-run the job for.
2. Click on the job you want to re-run.
3. Click on the "Re-run job" button in the top-right corner of the job log.

This will trigger a new workflow run with only the selected job re-run.

Feel free to ask, if you have any further doubts.

**Comment by ansiace** [comment]
Appreciate your reply @mdazfar2, but i don't see "Re-run all jobs" When I click on the three dots in the top-right corner from the dropdown menu.

**Reply by TamiyoucoVN** [reply]
Xin lỗi vì sự cố

**Comment by mdazfar2** [comment, marked as answer]
@ansiace The "Re-run all jobs" option is only available in the Actions tab of a GitHub repository, and only if there are previous workflow runs that have failed or been cancelled.

Here are the steps to find the "Re-run all jobs" option in the Actions tab of a GitHub repository:

1. Go to the repository's main page on GitHub.
2. Click on the "Actions" tab at the top of the page.
3. Find the failed or cancelled workflow run that you want to re-run, and click on it to open the workflow run page.
On the workflow run page, click on the "Re-run all jobs" button in the top-right corner of the page, next to the "Cancel workflow" button.
If there are no previous workflow runs that have failed or been cancelled, the "Re-run all jobs" option will not be available. In that case, you can manually trigger a new workflow run by pushing a new commit or by using the "Run workflow" button in the Actions tab.

**Comment by mdazfar2** [comment]
@ansiace
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by ansiace** [comment]
@mdazfar2 please accept my sincere apologies on delayed response. Thank you so much for your assistance.

**Reply by mdazfar2** [reply]
No worries at all! I'm here to help whenever you need it :)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ansiace ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by TamiyoucoVN** [reply]
Thanks mọi người
